src/components/AppToolbar.tsx

```diff
@@ -103,6 +105,7 @@ type AppToolbarInternalProps = {
   onSetMapState(mapState: MapState): unknown
   mapState?: MapState
   renderer?: string
+  onFileSelected(...args: unknown[]): unknown
```

Comment:
Don't we know the arguments type? This isn't great from a typescript perspective.